File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    count = 0
    num1 = a + b + c 
    num2 = (a * a) + (b * b) + (c * c)
    num3 = (a * a * a) + (b * b * b) + (c * c * c)
    cases = [ a == b, a == c, b == c]
    
    for i in cases :
        if i == 1:
            count += 1
        
    
    if count == 0 :
        return num1
    elif count == 1:
        return num1 * num2
    else :
        return num1 * num2 * num3
    
    

File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.swift
import Foundation

func solution(_ a:Int, _ b:Int, _ c:Int) -> Int {
    //pow가 되나?
    var count = 0
    let num1 = a + b + c 
    let num2 = (a * a) + (b * b) + (c * c)
    let num3 = (a * a * a) + (b * b * b) + (c * c * c)
    let cases = [ a == b, a == c, b == c]
    
    for i in cases {
        if i == true {
            count += 1
        } 
    }
    if count == 0 {
        return num1
    } else if count == 1{
        return num1 * num2
    } else {
        return num1 * num2 * num3
    }
    

}